Umbraco-Cloud/Release-Notes/2022-05-23-ReleaseNotes.md

```diff
@@ -8,9 +8,9 @@ _CDN Caching and Optimization settings + Improved Organization Invite Flow + Med
 - **Media Storage Files Top 50** - The Umbraco Cloud portal now offers greater transparency for which media files take up the most space in your cloud project's media storage.
 
 ## [CDN Caching and Optimization settings](https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/Manage-CDN-Caching/)
-In the Umbraco Cloud portal, you can now enable **caching** and ensure that the static resources such as javascript, CSS, and images are cached in the **Content Delivery Network** (CDN) used by Umbraco Cloud.
+In the Umbraco Cloud portal, you can now enable **caching**. This will ensure that static resources such as JavaScript, CSS, and images are cached in the **Content Delivery Network** (CDN) used by Umbraco Cloud.
 
-The feature enables you to set default settings for caching, cache TTL, minification of CSS, javascript, and HTML for your Cloud projects. If you prefer an alternative caching strategy to the default, for one or more hostnames, this can be done in **CDN Caching and Optimization** in the project's **Setting** menu.
+The feature enables you to set default settings for caching, cache TTL, minification of CSS, JavaScript, and HTML for your Cloud projects. If you prefer an alternative caching strategy to the default this can be done in **CDN Caching and Optimization** in the project's **Setting** menu.
```

Comment:
```suggestion
The feature enables you to set default settings for caching, cache Time To Live (TTL), minification of CSS, JavaScript, and HTML for your Cloud projects. If you prefer an alternative caching strategy to the default this can be done in **CDN Caching and Optimization** in the project's **Setting** menu.
```